Automattic/wp-calypso
# Importer Store: Remove redundant dispatcher-converter

#### Changes proposed in this Pull Request

* Remove redundant code (`importer/dispatcher-converter.js`)

#### Testing instructions

- Do a search on the code base before and after applying this branch, looking for `dispatcher-converter`.
- There should be no instances found in either

- Explore the import section after applying the branch, do an import etc.
- There should be no errors as the removed code was not referenced and not in use.
